server/entity/UserSettings.ts

```diff
@@ -20,24 +24,28 @@ export class UserSettings {
   @JoinColumn()
   public user: User;
 
-  @Column({ default: true })
-  public enableNotifications: boolean;
-
   @Column({ nullable: true })
-  public discordId?: string;
+  public region?: string;
 
```

Comment:
Please don't reorganize things unless it serves a purpose. This makes the PR really hard to read.